File: app/src/main/java/uk/lobsterdoodle/namepicker/AndroidLogWrapper.java
package uk.lobsterdoodle.namepicker;

import android.util.Log;

public class AndroidLogWrapper implements Logger {
    @Override
    public void v(String tag, String msg) {
        Log.v(tag, msg);
    }

    @Override
    public void d(String tag, String msg) {
        Log.d(tag, msg);
    }

    @Override
    public void i(String tag, String msg) {
        Log.i(tag, msg);
    }

    @Override
    public void w(String tag, String msg) {
        Log.w(tag, msg);
    }

    @Override
    public void e(String tag, String msg, Throwable ex) {
        Log.e(tag, msg, ex);
    }

    @Override
    public void wtf(String tag, String msg, Throwable ex) {
        Log.wtf(tag, msg, ex);
    }
}

File: app/src/main/java/uk/lobsterdoodle/namepicker/AndroidLogWrapper.kt
package uk.lobsterdoodle.namepicker

import android.util.Log

class AndroidLogWrapper : Logger {
    override fun v(tag: String, msg: String) { Log.v(tag, msg) }

    override fun d(tag: String, msg: String) { Log.d(tag, msg) }

    override fun i(tag: String, msg: String) { Log.i(tag, msg) }

    override fun w(tag: String, msg: String) { Log.w(tag, msg) }

    override fun e(tag: String, msg: String, ex: Throwable) { Log.e(tag, msg, ex) }

    override fun wtf(tag: String, msg: String, ex: Throwable) { Log.wtf(tag, msg, ex) }
}
